modules/logging-create-loki-cr-console.adoc

```diff
@@ -0,0 +1,83 @@
+= Creating a LokiStack custom resource by using the web console
+
+You can create a `LokiStack` custom resource (CR) by using the {product-title} web console.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the {product-title} web console.
+* You installed the {loki-op}.
+
+.Procedure
+
+. Go to the *Operators* -> *Installed Operators* page. Click the *All instances* tab.
+
+. From the *Create new* drop-down list, select *LokiStack*.
+
+. Select *YAML view*, and then use the following template to create a `LokiStack` CR:
+// tag::pre-5.9[]
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging
+spec:
+  size: 1x.small # <2>
+  storage:
+    schemas:
+    - version: v12
+      effectiveDate: '2022-06-01'
+    secret:
+      name: logging-loki-s3 # <3>
+      type: s3 # <4>
+      credentialMode: static #
+  storageClassName: <storage_class_name> # <5>
+  tenants:
+    mode: openshift-logging
+----
+<1> Use the name `logging-loki`.
+<2> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<3> Specify the secret used for your log storage.
+<4> Specify the corresponding storage type.
+<5> Enter the name of a storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed by using the `oc get storageclasses` command.
+// end::pre-5.9[]
+
+// tag::5.9[]
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging
+spec:
+  size: 1x.small # <2>
+  storage:
+    schemas:
+      - effectiveDate: '2023-10-15'
+        version: v13
+    secret:
+      name: logging-loki-s3 # <3>
+      type: s3 # <4>
+      credentialMode: # <5>
+  storageClassName: <storage_class_name> # <6>
+  tenants:
+    mode: openshift-logging
+----
+<1> Use the name `logging-loki`.
+<2> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<3> Specify the secret used for your log storage.
+<4> Specify the corresponding storage type.
+<5> Optional field, {logging} 5.9 and later. Supported user configured values are as follows: `static` is the default authentication mode available for all supported object storage types using credentials stored in a Secret. `token` for short-lived tokens retrieved from a credential source. In this mode the static configuration does not contain credentials needed for the object storage. Instead, they are generated during runtime using a service, which allows for shorter-lived credentials and much more granular control. This authentication mode is not supported for all object storage types.
```

Comment:
Same for this

credentialMode can also have the value of token-cco which is the default value when Loki is running on managed auth/STS mode on STS/WIF clusters from Logging 5.9 onwards